.github/actions/checkout/action.yml

```diff
@@ -0,0 +1,154 @@
+name: 'Checkout'
+description: 'Checks out Electron and stores it in the AKS Cache'
+inputs:
+  generate-sas-token:
+    description: 'Whether to generate and persist a SAS token for the item in the cache'
+    required: false
+    default: 'false'
+runs:
+  using: "composite"
+  steps:
+  - name: Set GIT_CACHE_PATH to make gclient to use the cache
+    shell: bash
```

Comment:
I don't think we need to specify bash in every step? is there a specific reason we're doing this

Reply:
Composite Actions require a `shell` specification, errors out without it being explicit